test/unbox-with-cache.js

```diff
@@ -111,3 +111,27 @@ tape('shared mutable state (source)', (t) => {
     )
   })
 })
+
+tape('shared mutable state (active)', (t) => {
+  const ssb = createSsb(`shared-mutable-state-${Date.now}`, {}, [require('ssb-private1')])
+
+  ssb.publish({ type: 'test', recps: [ssb.id]}, (err) => {
+    t.error(err)
+
+    pull(
+      ssb.createUserStream({ id: ssb.id, reverse: true, limit: 1, private: true }),
+      pull.collect((err, [privateMessageA]) => {
+        t.error(err)
+        const copy = cloneDeep(privateMessageA)
+        pull(
+          ssb.createUserStream({ id: ssb.id, reverse: true, limit: 1, private: true }),
+          pull.collect((err, [privateMessageB]) => {
+            privateMessageA.value = 'this should not effect privateMessageB'
+            t.deepEqual(copy, privateMessageB, 'active tampering should not mutate variables')
```

Comment:
further we can check they don't have the same reference I think

Reply:
this test is a little unclear to me. I don't know what passive / active means.

Also
Think you could check `copy` against `privateMessageA` later, should be the same (deepEqual).
I think you should check `privateMessageA` against `privateMessageB` - we expect deepEqual ... what do we expect about equal ..should the references be the same?

Reply:
Oh ignore me, reading tests in editor they make complete sense.
I just modified the test titles to be clearer (I think)